app/views/dashboard/show.html.haml

```diff
@@ -1,7 +1,7 @@
 - title t('homepage.title')
 
 .alert.alert-dark.mb-0.text-center
-  We’re beyond proud to launch our <strong>2023</strong> Impact Report, which you can read here - <a href="https://impact-report.codebar.io/" class="text-dark" target="_blank"">impact-report.codebar.io</a>
+  We’re beyond proud to launch our <strong>2024</strong> Impact Report, which you can read here - <a href="https://impact-report.codebar.io/" class="text-dark" target="_blank"">impact-report.codebar.io</a>
```

Comment:
```suggestion
  We’re beyond proud to launch our <strong>#{Date.today.year}</strong> Impact Report, which you can read here - <a href="https://impact-report.codebar.io/" class="text-dark" target="_blank"">impact-report.codebar.io</a>
```

Instead of hardcoding the year we can use this Date method to keep it up to date between years.